<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected File:
....................</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>66</y>
      <width>101</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>Start 
Process</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>20</width>
      <height>301</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alias</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Diffuse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Specular</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ambient</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <widget class="QMenu" name="menuHakk_nda">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionOrhan_Gazi_Hafif"/>
    <addaction name="actionLokman_Dursun"/>
    <addaction name="actionAhmet_Saydam"/>
    <addaction name="actionTe_ekk_rler_pek_Abas_kele_Turgut"/>
   </widget>
   <widget class="QMenu" name="menuKaynak_Dosya_Se_in">
    <property name="title">
     <string>Select Stage File</string>
    </property>
    <addaction name="actionC0"/>
    <addaction name="actionC1"/>
    <addaction name="actionAsdf"/>
    <addaction name="actionQwert"/>
    <addaction name="actionT1"/>
    <addaction name="actionC5"/>
    <addaction name="separator"/>
    <addaction name="actionT0"/>
    <addaction name="actionT1_2"/>
    <addaction name="actionT2"/>
    <addaction name="actionT3"/>
    <addaction name="separator"/>
    <addaction name="actionBilgisayar_n_zdan_Se_in"/>
   </widget>
   <addaction name="menuKaynak_Dosya_Se_in"/>
   <addaction name="menuHakk_nda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionC0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C0</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionC1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C1</string>
   </property>
  </action>
  <action name="actionAsdf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C2</string>
   </property>
  </action>
  <action name="actionQwert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>C3</string>
   </property>
  </action>
  <action name="actionOrhan_Gazi_Hafif">
   <property name="text">
    <string>Orhan Gazi Hafif</string>
   </property>
  </action>
  <action name="actionLokman_Dursun">
   <property name="text">
    <string>Lokman Dursun</string>
   </property>
  </action>
  <action name="actionAhmet_Saydam">
   <property name="text">
    <string>Ahmet Saydam</string>
   </property>
  </action>
  <action name="actionTe_ekk_rler_pek_Abas_kele_Turgut">
   <property name="text">
    <string>Special Thanks: İpek Abasıkeleş Turgut</string>
   </property>
  </action>
  <action name="actionT1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>C4</string>
   </property>
  </action>
  <action name="actionC5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>C5</string>
   </property>
  </action>
  <action name="actionT0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>T0</string>
   </property>
  </action>
  <action name="actionT1_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>T1</string>
   </property>
  </action>
  <action name="actionT2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>T2</string>
   </property>
  </action>
  <action name="actionT3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>T3</string>
   </property>
  </action>
  <action name="actionBilgisayar_n_zdan_Se_in">
   <property name="text">
    <string>Select From Other Location...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
